### ISSUE TITLE
hard to debug

### ISSUE THREAD

**Issue by wuhaitao93** [gilad-rubin/hypster#8]
Although this modular method is more convenient to maintain, how to debug the components defined by configuration files?
 The breakpoints in config file can't be hit, so it is difficult to use in new project.

**Comment by gilad-rubin** [comment]
That's a good question. The short answer is no - it's not possible at the moment.
I'll give it some thought and reply back.

Thanks!

**Comment by gilad-rubin** [comment]
In the meantime, I think the best option would be to utilize automatic logging.
Currently the logging is under-developed, but I can improve it so that it'll be possible to see the selected values for each HP.

Will that help?

**Comment by wuhaitao93** [comment]
Yes, I guess automatic logging is a good solution, like the one used in haystack's pipeline. Thanks for the reply!
